docs/source/science/properties/morphology.rst

```diff
@@ -3,4 +3,97 @@
+
+
+Nanomatch Multiscale Workflow for Morphology
+--------------------------------------------
+
+In the DiaDEM Platform, the Nanomatch Multiscale Workflow for morphology is employed (:ref:`science_calculators_morphology`), which includes:
+
+- Molecule parametrization,
+- Morphology deposition using a Monte-Carlo protocol mimicking physical deposition (*DEPOSIT protocol* [1]_),
+- Analysis of the generated morphology using various computational tools,
+- Calculation of key morphological properties such as radial distribution function, mass and number density, mean distance to the nearest neighbours.
+
+The morphology can be downloaded under the name `structure.cml` along with other files (:ref:`science_calculators_morphology_files`).
+
+The morphological properties, which are computed within the calculator  :ref:`science_calculators_morphology` are explained in the Table below.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+     - Units
+     - Value (example)
+     - Additional Information
+   * - average_neighbors
+     - Average distance to neighboring molecules
+     - Å
+     - 17.6
+     - Average distance to neighboring molecules
+   * - mass_density
+     - Mass per unit volume
+     - g/cm³
+     - 1.14
+     - Standard deviation (`std`): 0.01 (example value)
+   * - molecular_volume
+     - Volume occupied by a single molecule
+     - nm³
+     - 0.23
+     -
+   * - number_density
+     - Number of molecules per unit volume
+     - 1/cm³
+     - 4.36e+21
+     - Standard deviation (`std`): 9.9e+19 (example value)
+   * - rdf_first_peak
+     - Position of the first peak in the radial distribution function
+     - Å
+     - 4.921630094043887
+     - Indicates the most probable intermolecular distance
+
+
+Using existing or generated morphology, the determination of secondary properties that depend on the morphology, such as charge carrier mobility, ionization potentials (IP), and electron affinities (EA) as possible as well.
+For example, see :ref:`science_properties_mobility`.
+
+Although this workflow integrates extensive experience and advanced simulation methods, including quantum chemistry (QM), molecular mechanics (MM), Monte-Carlo, the only input required from the user is the structure of the molecule, for example, InChI or another identifier. This makes it accessible to all users, regardless of their background.
+
+The DiaDEM project allows you to compute detailed morphological characteristics and secondary properties for any single-component organic semiconductor by simply specifying the molecule in the form of an InChI key.
+
+
+Other Workflows that Provide Morphology
```

Comment:
I would keep this more precise in the DiaDEM phrasing - it is only chance that in this case the calculator and the property are named similarly. Typically, a property is just called HOMO and the calculator is called FranzGenericVoodoo. So my idea is to place a section in all properties as follows: 

Calculators providing morphology/HOMO/LUMO/PropertyXYZ
-----------------------------------
* Calculator A
* Calculator B


Also should we put it more prominently near the top of the property description so the user can directly refer to the method as described in the different calculators, when he knows what a property is but just wants to check different available methods for details?

Reply:
Yes, this is a good idea.
How to we name our two claculators, one for mobility one for the morphology?

Electron_and_Hole_Mobility_Nanomatch
Mobility_Nanomatch
Mobility_NM

Morphology_Nanomatch
Morphology_NM

any conventions in naming?

Reply:
They do not exist so far as the images for productive runs, we can name them as we want

Reply:
I am fine with either suggestion. Keep in mind that you can have spaces and it should be human readable ;-) People not used to programming may find the "_" funny.

Reply:
I put the Workflows right after the title for a quick reference.